google/ggrc-core
# GGRC-4074 Disallow creating Global Custom Attributes with Person type

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [ ] #9075 

# Issue description

Remove possibility to add global custom attributes with Person type for any object.

# Steps to test the changes

1. Open Admin page
2. Try to add custom attribute with Person type

Expected result: there's no Person type in attribute's type dropdown.

Note: local custom attributes with Person type for Assessment should not be removed.

# Solution description

Remove `Map:Person` option from CustomAttributable model.
Clean up related code.

# Sanity checklist




# PR Review checklist
